File: csharp/MapleStory.OpenAPI/Src/Dto/CharacterCashItemEquipmentOptionDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 캐릭터 캐시 장비 옵션
    /// </summary>
    public class CharacterCashItemEquipmentOptionDTO
    {
        /// <summary>
        /// 옵션 타입
        /// </summary>
        [JsonProperty("option_type")]
        public string OptionType { get; set; }

        /// <summary>
        /// 옵션 값
        /// </summary>
        [JsonProperty("option_value")]
        public string OptionValue { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/msea/dto/character/CharacterCashItemEquipmentOptionDTO.java
package dev.spiralmoon.maplestory.api.msea.dto.character;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * Cash equipment option
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class CharacterCashItemEquipmentOptionDTO implements dev.spiralmoon.maplestory.api.common.dto.character.CharacterCashItemEquipmentOptionDTO {

    /**
     * Option type
     */
    @JsonProperty("option_type")
    private String optionType;

    /**
     * Option value
     */
    @JsonProperty("option_value")
    private String optionValue;
}
